### ISSUE TITLE
Cannot execute build_membership.py

### ISSUE THREAD

**Issue by fredrikaverpil** [mottosso/Qt.py#220]
@mottosso I noticed I get this when I try to execute the script:

```bash
# Python 3.5.2 on Ubuntu 16.04 Xenial
$ python build_membership.py 
  File "build_membership.py", line 4
    def build_membership():
SyntaxError: import * only allowed at module level
```

I wanted to run this to compare the current members dict against the output using a freshly compiled PySide2 from the 5.9 branch.

Is this something you've stumbled upon before?

**Comment by mottosso** [comment]
Yeah, I did stumble upon this. You can't do that in Python 3. You'll notice I'm only testing memberships [under Python 2.7](https://github.com/mottosso/Qt.py/blob/master/Dockerfile-py2.7), this is the reason.

I never found a way around it and instead assume memberships are identical across versions of Python.

**Comment by fredrikaverpil** [comment]
Also, I think QtOpenGL could now be included.

```python
>>> from PySide2 import QtOpenGL
>>> print(QtOpenGL)
<module 'PySide2.QtOpenGL' from '/usr/local/lib/python2.7/dist-packages/PySide2/QtOpenGL.x86_64-linux-gnu.so'>
>>> print(dir(QtOpenGL))
['QGL', 'QGLBuffer', 'QGLColormap', 'QGLContext', 'QGLFormat', 'QGLFramebufferObject', 'QGLFramebufferObjectFormat', 'QGLPixelBuffer', 'QGLShader', 'QGLShaderProgram', 'QGLWidget', '__doc__', '__file__', '__name__', '__package__']
>>> 
```

It looks like the whole `from PySide2 import *` was because there was a discrepancy here where `QtOpenGL` was part of `__all__` but did not exist as a module?

**Comment by fredrikaverpil** [comment]
Ok, I looked into it. PR coming up.

**Comment by fredrikaverpil** [comment]
Fixed in e705a7ab4fc5f437d78269d874f9956048aa6d81
